mne/source_estimate.py

```diff
@@ -700,6 +700,13 @@ def __neg__(self):  # noqa: D105
     def __pos__(self):  # noqa: D105
         return self
 
+    def __abs__(self):
+        """Take the absolute value."""
```

Comment:
This is defined in the `_BaseSourceEstimate` class. Not all source estimated have components.